packages/clerk-js/src/ui/primitives/Button.tsx

```diff
@@ -164,10 +165,7 @@ const Button = React.forwardRef<HTMLButtonElement, ButtonProps>((props, ref) =>
   return (
     <button
       {...applyDataStateProps(rest)}
-      // Explicitly remove type=submit or type=button
-      // to prevent global css resets (eg tailwind) from affecting
-      // the default styles of our components
-      type={undefined}
+      type={rest.type}
```

Comment:
IIRC type needs to be explicitly set to `undefined` to avoid framework resets. `rest.type` should be undefined if not overridden by the caller, but we should double check

(I'd also leave the comment in place as we've already been bitten by this once)

Reply:
Also, I think its easy to test by mounting a SignIn/SignUp component in a next + tailwind project and checking whether the form Continue button is broken